openapiGenerator.go

```diff
@@ -72,6 +72,7 @@ var specialSoloTypes = map[string]openapi3.Schema{
 	"google.protobuf.DoubleValue": *openapi3.NewFloat64Schema().WithNullable(),
 	"google.protobuf.Int32Value":  *openapi3.NewIntegerSchema().WithNullable().WithMin(math.MinInt32).WithMax(math.MaxInt32),
 	"google.protobuf.UInt32Value": *openapi3.NewIntegerSchema().WithNullable().WithMin(0).WithMax(math.MaxUint32),
+	"google.protobuf.UInt64Value": *openapi3.NewIntegerSchema().WithNullable().WithMin(0).WithMax(math.MaxUint64),
```

Comment:
Looks good! Should we also add Int64Value support while we're here?

Reply:
Why not!